editor/src/core/model/project-file-utils.ts

```diff
@@ -230,6 +233,24 @@ export function getHighlightBoundsFromParseResult(
   )
 }
 
+export function getHighlightBoundsForProject(
```

Comment:
If this function ends up being too costly, we can instead add the file path to the highlight bounds during the parsing at no extra cost.

Reply:
I don't think it should be too costly, albeit it is running for each selected and highlighted view.